Assets/Scripts/Player/PlayerController.cs

```diff
@@ -31,4 +47,20 @@ public class PlayerController : MonoBehaviour {
         // TODO: Maybe set targetPosition to currentPosition when the player is stopped. Then, the player
         // won't continually try to reach an unreachable targetPosition when blocked by a collider.
     }
+    public void takeDamage(int damage)
+    {
+        if (currentHealth > 0)
+        {
+            currentHealth -= damage;
+            healthBar.SetHealth(currentHealth);
+        }      
+    }
+    public void heal(int heal)
+    {
+        if (currentHealth < maxHealth)

```

Comment:
- [ ] If the player is at 90 health and gets healed for 20 health, `currentHealth` will go above `maxHealth`. If the player is at 10 health and gets damaged for 20 health, `currentHealth` will go below 0. These errors can be fixed by doing the increment or decrement first and then setting `currentHealth` to the limit if it goes past the limit.